### DISCUSSION TITLE
Build Android Projects with Visual Interface

### DISCUSSION THREAD

**Discussion by Shirley-yhm** [o3de/o3de#14548] [General]
Users can generate Android projects by scripts in cmd now, which may not be user friendly. Have you considered providing a visual interface to simplify users' operations just like what `Platform Settings` window does?

My team has done some work to this problem. In our design, users can set settings like the path of Gradle in a visual window instead of command line. Then they can choose to export or build&pack the Android project.

We want to contribute this feature to the community now. But I worry that it may conflict with `Platform Settings` window in function or design. Is there any document about `Platform Settings` window? Also, I wonder if you have any suggestions on this problem. Thanks :)

**Comment by thefranke** [comment]
Hey @Shirley-yhm !

Please write an RFC for this feature in https://github.com/o3de/sig-platform/issues . You can add images to show how it looks like, and how it ties into the rest of the engine. I'd also suggest to look at other RFCs in other sig repositories to get some inspiration how to write it. The SIG you write an RFC for (in this case sig-platform) will then review the feature, perhaps make some suggestions or request changes, after which you can begin the PR for your suggested feature.

**Reply by Shirley-yhm** [reply]
I got it. Thanks!

**Comment by Shirley-yhm** [comment]
RFC: https://github.com/o3de/sig-platform/issues/67
